src/hint/Hint.tsx

```diff
@@ -0,0 +1,78 @@
+import React, { Component } from 'react'
+import cc from 'classcat'
+import uniqueid from 'lodash.uniqueid'
+
+import { A11yProps } from '_utils/interfaces'
+import HintBubble, { HintBubblePosition } from './HintBubble'
+
+export interface HintProps extends A11yProps {
+  children: (a11yAttrs: A11yProps) => React.ReactNode
```

Comment:
Why is children a function? It'll be more consistant to just have a react node

Reply:
I need to pass some new props to the child, so it's cleaner this way to keep the typing and better for performances than React.clone(). Also it's more flexible so you can attach the a11y attribute to the right element.